ansible/roles/fanout/tasks/main.yml

```diff
@@ -1,43 +1,36 @@
-# Gather minigraph facts
+##############################################################################################
+### playbook to deploy the fanout swtich 
+### Use this playbook to deploy the VLAN configurations of fanout leaf switch in SONiC testbed
+### This playbook will run based on hardware flatform. Each fanout switch hardware type has its
+### own unique feature and configuration command or format. Unless you use the hardware swtich
+### specified in this playbook, you would need to come up with your own fanout switch deployment 
+### playbook
+################################################################################################
+
+- include_vars: group_vars/lab/lab.yml
+  
+
+  # Gather minigraph facts
 - name: Gathering lab graph facts about the device
   conn_graph_facts: host={{ inventory_hostname }}
   connection: local
   tags: always
 
-- block:
-  - name: prepare {{ sonicadmin_user }} password
-    set_fact: ansible_ssh_user={{ sonicadmin_user }} ansible_ssh_pass={{ sonicadmin_password }}
+- name: prepare fanout switch admin login info
+  set_fact: ansible_ssh_user={{ fanout_admin_user }} ansible_ssh_pass={{ fanout_admin_password }}
 
-  - name: setup fanout bash account
-    template: src=rc.eos.j2
-              dest=/mnt/flash/rc.eos
```

Comment:
I don't think this step is useful since we are logging through eosadmin, there is not use to re-set it.
I think this step should be a preconfigure step for eos switch before running this playbook.